src/main/java/com/solvd/pages/android/components/CartItemComponent.java

```diff
@@ -0,0 +1,20 @@
+package com.solvd.pages.android.components;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class CartItemComponent extends AbstractComponent {
+    @FindBy(xpath = ".//android.view.ViewGroup[@content-desc='test-Description']/android.widget.TextView[1]")
```

Comment:
it's a good practice to leave a blank line between the class declaration and the first field or method